RadarSDK/RadarLocationManager.m

```diff
@@ -1127,8 +1129,17 @@ - (void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *
 }
 
 - (void)locationManager:(CLLocationManager *)manager didChangeAuthorizationStatus:(CLAuthorizationStatus)status {
+    if (_firstPermissionCheck) {
+        _firstPermissionCheck = NO;
```

Comment:
```suggestion
    if (firstPermissionCheck) {
        firstPermissionCheck = NO;
```

Reply:
We need the `_` as this is an internal flag not defined in the header.